create/how_to/aws.md

```diff
@@ -15,23 +15,23 @@ The following guide will walk you through every step to deploy MeiliSearch in an
 
 After logging into your [AWS Console](https://aws.amazon.com/console), navigate to the **Compute** service. Then go to **EC2**, and finally open your **Instances** console.
 
-![Launch an instance from AWS console](/aws/01.launch-instances.png)
+![Page titled 'Instances'. Text in center of screen: You do not have any instances in this region](/aws/01.launch-instances.png)
```

Comment:
We do have the context in the the heading and text. The [article](https://supercooldesign.co.uk/blog/how-to-write-good-alt-text) that Tommy shared said don't repeat what's already on the page.

Reply:
That's a fair point. The same article does mention that it can be useful to point out when something is a screenshot, but if you think it's not necessary feel free to mark these comments as resolved.